README.md

```diff
@@ -5,21 +5,21 @@ Join us on slack [![Alt text](images/slack.jpeg)](https://launchpass.com/lensesi
 
 # Lenses for Apache Kafka
 
-Lenses offers SQL (for data browsing and Kafka Streams), Kafka Connect connector management, cluster monitoring and more.
+Lenses offers SQL (for data browsing and Kafka Streams), Kafka Connect connector management, cluster monitoring and
+more.
 
 You can find more on [lenses.io](http://www.lenses.io)
 
 # Stream Reactor
 
-
 ![Alt text](images/streamreactor-logo.png)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Flensesio%2Fstream-reactor.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Flensesio%2Fstream-reactor?ref=badge_shield)
 
-
 A collection of components to build a real time ingestion pipeline.
 
 ## Kafka Compatibility
-* Kafka 2.8 -> 3.3 (Confluent 2.8 -> 7.3) - Upcoming Stream Reactor 4.1.0
+
+* Kafka 2.8 -> 3.5 (Confluent 2.8 -> 7.5) - Stream Reactor 4.1.0+
```

Comment:
This line is wrong, no fault of your own.

Kafka 2.8 - 3.5 is equivalent to confluent platform **6.2** - 7.5

Reply:
resolved